worknenjoy/gitpay
# Defines notification messages to support internationalization

## Description

Define notifications messages and generates translation files.

## Changes

```
modified:   frontend/package.json
modified:   frontend/src/actions/messages/messages.js
modified:   frontend/src/translations/generated/br.json
modified:   frontend/src/translations/generated/en.json
```

## Issue

> Closes #705 

## Impacted Area

> Notifications throughout site

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Create / log user
- See the notification (login message successful)
- Logout (logout message not translated)
